README.md

```diff
@@ -23,6 +23,9 @@ The following parameters require updates:
  - `discovery.apikey` - An API Key can be obtained from the api-manager for the user who has access to post the API.
 An API key can be created by logging into the APIC Manager UI and selecting the "My API Keys" option under the profile icon from the top navigation bar. 
 The apikey will be added to a kubernetes secret as part of the deployment and then mounted on the collector deployment pod.  
+ - `discovery.processor` - The flag to enable or disable the processors. The [configmap](apidiscovery/templates/processor-configmap.yaml) enables the existing batch and memory_limiter processor when `enabled` value is given and will not provide any processors in other cases. The processors should be enabled if the open telemetry collector needs to send a trace with more than 11000 spans. The available batch and memory limiter processors are configurable according to the requirements. <br /> &nbsp;
+Refer - [Batch processor](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md#batch-processor) and [Memory Limiter processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor#configuration)<br />
+Keeping the `filter` in the [`service.pipelines.traces.processors`](apidiscovery/templates/processor-configmap.yaml#L21) while enabling processors is recommended, as we have a built-in filter processor that enhances the collector's performance.
```

Comment:
processors is required, as we have a built-in filter processor that enhances the collector's performance. If you neglect to include the filter processor it will degrade the quality of the collected traffic as well as the performance of the collector.